package.json

```diff
@@ -343,7 +343,8 @@
       "simulator": {
         "type": "ios.simulator",
         "device": {
-          "type": "iPhone 12 Pro Max"
+          "type": "iPhone 11 Pro Max",
+          "os": "13.7"
```

Comment:
why 13?

Reply:
because this one works and is available. This is not changing anything tbh, we're still building with the newest xcode.

Reply:
yeah all good was jw